recipes/attributes.rb

```diff
@@ -0,0 +1,21 @@
+#
+# Cookbook:: osl-postfix
+# Recipe:: attributes
+#
+# Copyright:: 2019, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+node['osl-postfix']['main'].each do |key, value|
+  node.default['postfix']['main'][key] = value
+end
```

Comment:
I don't think this needs its own recipe and just can be included in the default recipe in the appropriate location.

Reply:
I've refactored the layout not to use a separate recipe.